packages/data-store/src/types/contact/contact.ts

```diff
@@ -13,6 +14,8 @@ export type Party = {
   physicalAddresses: Array<PhysicalAddress>
   contact: Contact
   partyType: PartyType
+  //todo: add logic for integrating branding in here. for now we're just using the type
```

Comment:
Please clarify that message way more. I cannot determine what you actually mean with it

Reply:
modified the comment and clarified it more based on the discussion that we had